OpenRA.Mods.Common/Traits/LintAttributes.cs

```diff
@@ -13,7 +13,7 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
-	[AttributeUsage(AttributeTargets.Field)]
+	[AttributeUsage(AttributeTargets.Field | AttributeTargets.Property)]
```

Comment:
Should we update `AttributeUsage` on all `*ReferenceAttribute`s? to allow both fields and properties since they're now supported? Might be worth declaring a constant and reusing it for all such attributes.

Reply:
Actually I see that this is not the case and we'd have to update various lint checks to support properties.

Which might be nice so you can use properties with all lint checks without finding sometimes that isn't allowed, but that's a job for a different PR really if we did decide on such a thing.